### PR TITLE
Linkify even more things

### PR DESCRIPTION
**Pull request by lidavidm**

### CODE REVIEW

File: classes/CS2800.md [hunk 1]
```diff
@@ -62,13 +62,14 @@ Regardless, Bailey gave a noticeable effort to help out his students, offering e
 ## Past Offerings
 
 {:.past-offerings}
-| Semester | Time | Professor | Median Grade | Course Page |
```
Comment (pniedzielski) [inline classes/CS2800.md]: ?

Comment (lidavidm) [inline classes/CS2800.md]: Emacs ate a space at the end of the line.

Comment (pniedzielski) [inline classes/CS2800.md]: gotcha

**Comment by pniedzielski** [comment]
👍

**Pull request merged**